disnake/__main__.py

```diff
@@ -58,7 +58,8 @@ def core(parser, args):
         show_version()
 
 
-_bot_template = """#!/usr/bin/env python3
+_bot_template = """
```

Comment:
Why was this changed? now when a user makes a bot with this command there’s an empty space at the top.

Reply:
whoops, result of a regex, thanks for the prompt review!

Reply:
That's not really a problem, we can do a simple commit reverting that, I just don't like how docstrings look now  :/

Reply:
This was changed in e07d030d62ce13de428a7c67bb823b5051b66709, thanks for the review.